src/dataflow/analysis/flow-graph.ts

```diff
@@ -33,8 +33,8 @@ export class FlowGraph {
   /** Maps from particle name to node. */
   readonly particleMap: Map<string, ParticleNode>;
 
-  /** Maps from HandleConnectionSpec to HandleNode. */
-  private readonly handleSpecMap: Map<HandleConnectionSpec, HandleNode> = new Map();
+  /** Maps from HandleConnectionSpecIntf to HandleNode. */
+  private readonly handleSpecMap: Map<HandleConnectionSpecIntf, HandleNode> = new Map();
```

Comment:
Could we use `Interface` over `Intf`? Maybe it's too verbose but I think it improves readability.

Reply:
Sure. Done.